docs/NEW_LANGUAGE_FEATURE_INTEGRATION_PROCESS.md

```diff
@@ -36,18 +40,18 @@ on existing checks.
 
 **Examples**:
 
-- When the `RECORD_DEF` token was introduced, it was reasonable to look at all checks
-  that had `CLASS_DEF` in their acceptable tokens.
+- **Check updates**: When the `RECORD_DEF` token was introduced,
+ it was reasonable to look at all checks that had `CLASS_DEF` in their acceptable tokens.
 
-- When the `TEXT_BLOCK_CONTENT` token was introduced, it was reasonable to look at all checks
-  that had `STRING_LITERAL` in their acceptable tokens.
+- **Check updates**: When the `TEXT_BLOCK_CONTENT` token was introduced,
+  it was reasonable to look at all checks that had `STRING_LITERAL` in their acceptable tokens.
 
-- When switch expressions were introduced, it was reasonable to look at all checks
-  that deals with expressions because `LITERAL_SWITCH` may now fall under `EXPR` token.
+- **Check updates**: When switch expressions were introduced, it was reasonable to look at all
+  checks that deals with expressions because `LITERAL_SWITCH` may now fall under `EXPR` token.
   This requires analyzing the impact of this new child token on existing checks,
   such as `InnerAssignmentCheck`
```

Comment:
Not for this line, but why don't we have some new check examples in here? Surely there are some cases where we found a check that could have supported some new token, but decided to make a new check instead.

Reply:
done